src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.test.ts

```diff
@@ -178,6 +160,29 @@ const expectedTradeQuoteFoxToEth: TradeQuote<KnownChainIds.EthereumMainnet> = {
   accountNumber: 0,
 }
 
+const expectedTradeQuoteUsdcToXdai: TradeQuote<KnownChainIds.GnosisMainnet> = {
+  rate: '4.996e-17',
```

Comment:
ditto scientific notation

Reply:
@lmyslinski do we want to tackle this or resolve?

Reply:
https://github.com/shapeshift/web/pull/4595/commits/75aa0d252992d66fedd3ba0c994573f1bd9fa69e along with https://github.com/shapeshift/web/pull/4595/commits/de017bf532321dc4c4f2b22686e10354711da347